From f2be338e25918faa544ef1b97f22a9dd517037a9 Mon Sep 17 00:00:00 2001
From: Michal Vasilek <michal.vasilek@nic.cz>
Date: Thu, 2 Jun 2022 13:59:09 +0200
Subject: [PATCH] apr-util: use libmariadbclient for dbd_mysql

libmariadb is in the upstream mariadb package, but not in
turrispackages mariadb
---
 libs/apr-util/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libs/apr-util/Makefile b/libs/apr-util/Makefile
index 8938acd..182471e 100644
--- a/libs/apr-util/Makefile
+++ b/libs/apr-util/Makefile
@@ -160,7 +160,7 @@ endef
 
 $(eval $(call BuildPackage,libaprutil))
 $(eval $(call Package/libaprutil/Driver,crypto_openssl,OpenSSL,+libopenssl))
-$(eval $(call Package/libaprutil/Driver,dbd_mysql,MySQL,+libmariadb))
+$(eval $(call Package/libaprutil/Driver,dbd_mysql,MySQL,+libmariadbclient))
 $(eval $(call Package/libaprutil/Driver,dbd_odbc,ODBC,+unixodbc))
 $(eval $(call Package/libaprutil/Driver,dbd_pgsql,PostgreSQL,+libpq))
 $(eval $(call Package/libaprutil/Driver,dbd_sqlite3,SQLite3,+libsqlite3))
-- 
2.36.1

